numpy-stubs/__init__.pyi

```diff
@@ -349,6 +354,7 @@ class Array(Protocol[_DType]):
     def __array__(self) -> Union[ndarray[_DType], Sequence[Sequence[_DType]]]: ...
 
 _ArrayLike = Union[Array[_DType], Sequence[_DType]]
+_Coercable = Union[_ArrayLike, _DTypeObj]
```

Comment:
Added as `atleast_2d` can have inputs of `_ArrayLike`, or values that are not `_ArrayLike` , but that can be coerced into an array

Reply:
btw, I didn't think you could coerce a str, but apparently you can 🤷‍♂️